proof/refine/ARM/Syscall_R.thy

```diff
@@ -461,10 +460,10 @@ lemma pinv_corres:
         \<comment> \<open>domain cap\<close>
         apply (clarsimp simp: invoke_domain_def)
         apply (rule corres_guard_imp)
-          apply (rule corres_split_deprecated [OF _ set_domain_setDomain_corres])
+          apply (rule corres_split_deprecated [OF _ setDomain_corres])
             apply (rule corres_trivial, simp)
            apply (wp)+
-         apply (clarsimp+)[3]
+         apply ((clarsimp | fastforce)+)[3]
```

Comment:
Out of curiosity, could this just be `fastforce`?

Reply:
Strangely, no.